cvd/debian/control

```diff
@@ -0,0 +1,27 @@
+Source: cuttlefish-cvd
+Maintainer: Cuttlefish Team
+Section: misc
+Priority: optional
+Build-Depends: config-package-dev,
```

Comment:
#395 includes a list of packages that were requirements to get it to build on a workflow runner.

`clang pkg-config meson libfmt-dev libgflags-dev libjsoncpp-dev protobuf-compiler android-libboringssl-dev libgtest-dev libcurl4-openssl-dev libprotobuf-c-dev libgoogle-glog-dev libxml2-dev uuid-dev`

Looks like `clang pkg-config meson` are missing from the list here? Not sure if those all are provided by `config-package-dev`. There's also `libprotobuf-dev` vs `libprotobuf-c-dev`.

Reply:
done